src/LightGraphsExtras.jl

```diff
@@ -4,10 +4,12 @@ export Matching
 export CombinatorialAdjacencies
 export Datasets
 export Community
+export Interdiction
 
 include("matching/matching.jl")
 include("spectral/spectral.jl")
 include("datasets/Datasets.jl")
 include("community/detection.jl")
+include("interdiction/Interdiction.jl")
```

Comment:
OK. I was not sure since I was starting a new submodule.

Do you want me to changeDatasets.jl too? It is not a hard task!